src/locale/types.js

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+export type DatepickerLocaleT = {
+  ariaLabel: string,
+};
+
+export type LocaleT = {|
+  datepicker: DatepickerLocaleT,
```

Comment:
Should we be overriding the English locale and keep all `LocaleT` props optional?

Reply:
I mean user can do that but it should be a `deepMerge` and seems nice to do it by default.

Reply:
that makes perfect sense 👍